app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>oka</h1>
```

Comment:
If we don't have anything to show on root path or any page we can use some dummy content or template.